app/javascript/elm/src/Main.elm

```diff
@@ -264,7 +280,7 @@ update msg model =
         ToggleSessionSelectionFromAngular maybeId ->
             case maybeId of
                 Just id ->
-                    ( model, SelectedSession.fetch model.selectedSensorId model.page id GotSession )
+                    ( model, SelectedSession.fetch model.selectedSensorId model.page id (RemoteData.fromResult >> GotSession) )
```

Comment:
what do you think of using `<<` to make it read better? (same for other places)

Reply:
You'd prefer ` GotSession << RemoteData.fromResult` instead of `RemoteData.fromResult >> GotSession` ?

Reply:
this style was suggested as possibly better / easier to read in Elm in Action and at first I didn't know why but now it does make sense - I'm passing the result of the function to another one like I would pass an argument, so this order is more comfortable to read, I think. wdyt?

Reply:
I'm all in for that

Reply:
I'll be doing it from now on and refactor whenever I see composition